app/helpers/admin/enterprises_helper.rb

```diff
@@ -71,6 +71,12 @@ def enterprise_attachment_removal_panel_id
       "#{enterprise_attachment_removal_panel}_panel"
     end
 
+    def enterprise_sells_options
+      Enterprise::SELLS.map do |s|
+        [I18n.t("admin.enterprises.admin_index.sells_options.#{s}"), s]
+      end
```

Comment:
You don't need to change it but just for future reference, you could write it like this:
```suggestion
      scope = "admin.enterprises.admin_index.sells_options"
      Enterprise::SELLS.map do |s|
        [I18n.t(s, scope:), s]
      end
```